import/README.md

```diff
@@ -31,17 +53,18 @@ Peak memory pressure in megabytes = MB
 
 Not possible = np
```

Comment:
Is it better to go with classic notation [N/A](https://en.wikipedia.org/wiki/N/A)?

Reply:
Yeah, great suggestion.